fs/layer/node.go

```diff
@@ -244,6 +244,26 @@ func (fs *fs) inodeOfStatFile() uint64 {
 	return (uint64(fs.baseInode) << 32) | 2 // reserved
 }
 
+// basicTelemetry wraps informational FUSE telemetry operations.
+func (fs *fs) basicTelemetry(ctx context.Context, operationName string, path string) {
```

Comment:
Hmm, can we brainstorm this a little more. This function is logging and incrementing a counter. Maybe `logAndIncrementOperation`?